apps/condo/domains/common/utils/JSON.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Replaces cyclic links within object for JSON.stringify to be able to serialize object properly
+ * usage: JSON.stringify(obj, getCycleReplacer(), 2)
+ * @returns {function(*, *=): *}
+ */
+const getCycleReplacer = () => {
```

Comment:
it's not a part of this PR. move to another

Reply:
ok